test/modules/load-gen/PriorityMessageSender.cs

```diff
@@ -28,16 +28,25 @@ class PriorityMessageSender : LoadGenSenderBase
             : base(logger, moduleClient, batchId, trackingId)
         {
             this.PriorityString = Settings.Current.Priorities.Expect(() =>
-                new ArgumentException("PriorityMessageSender must have 'priorities' environment variable set to a valid list of string delimited by ';'"));
+                new ArgumentException("PriorityMessageSender must have 'priorities' environment variable set to a valid list of strings delimited by ';'"));
+            this.TtlString = Settings.Current.Ttls.Expect(() =>
```

Comment:
Both TtlString and PriorityString should be converted to collection type in Settings class.  Do we have use case that ttl string is better than collection type, e.g. array or list?